pkg/util/deployment/deployment.go

```diff
@@ -28,8 +28,13 @@ import (
 	podutil "k8s.io/kubernetes/pkg/util/pod"
 )
 
+const (
+	// The revision annotation of a deployment's replication controllers which records its rollout sequence
+	RevisionAnnotation = "deployment.kubernetes.io/revision"
```

Comment:
asked about this in https://github.com/kubernetes/kubernetes/pull/19686#discussion_r50973865 but I think this is the correct place